x/interchaintxs/keeper/keeper.go

```diff
@@ -69,8 +65,8 @@ func NewKeeper(
 
 		icaControllerKeeper: icaControllerKeeper,
 		scopedKeeper:        scopedKeeper,
-		wasmKeeper:          wasmKeeper,
-		sudoHandler:         sudo.NewSudoHandler(wasmKeeper, types.ModuleName),
+		contractManager:     contractManager,
+		sudoHandler:         contractManager.NewSudoHandler(types.ModuleName),
```

Comment:
such pair instantiations are the reason why I've suggested to merge these two structures above, I find them not pretty :)

Reply:
and I don't think this solves the second issue of the task this PR is related to:

> passing wasmKeeper into interchainTxKeeper and also creating sudoHandler inside.

Reply:
The problem here is that I need to pass `ModuleName` inside it. So I need to instantiate new instance of `SudoHandler` inside keeper. I will try to find another, more suitable approach, but at the moment this is the most convenient way.